glusterd2/volgen2/clustergraph.go

```diff
@@ -73,6 +75,9 @@ func clusterGraph(volfile *Volfile, dht *Entry, vol *volume.Volinfo, nodeid uuid
 
 		if parent != nil {
 			for brickIdx, b := range subvol.Bricks {
+				afrPendingXattr = append(afrPendingXattr, fmt.Sprintf("%s-client-%d", vol.Name, clientIdx))
+				clientIdx++
```

Comment:
Will clientIdx++ always yield the same value when you do add-brick etc? The value has to be persistent. For example, If I create a 1x3 and then remove the middle brick to make it 1x2, the value must still be
`option afr-pending-xattr testvol-client-0,testvol-client-2` and not
`option afr-pending-xattr testvol-client-0,testvol-client-1`

Not sure if persistent-xattrs for AFR are taken care of in gd2. See https://review.gluster.org/#/c/7155/ 's commit message.

Reply:
This is not yet considered in gd2, I will fix it. Since this issue can happen only when add/remove brick. Can this be addressed in next PR?

Reply:
I guess that should be fine.